RFC/0000-RFC-TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+- Start Date: 2019-08-15
+- RFC PR:
```

Comment:
What's the thought here, it seems unnecessary? Ultimately we should be able to get the commit hash from git history and the order of operations for updating this after the PR is created makes me think this should go away.